credentials/src/test/kotlin/web5/sdk/credentials/PresentationExchangeTest.kt

```diff
@@ -450,8 +450,8 @@ class PresentationExchangeTest {
 
       val selectedCreds = PresentationExchange.selectCredentials(listOf(vcJwt), pd)
 
-      assertEquals( 1, selectedCreds.size)
```

Comment:
err I wonder why my formatter is doing this.

for web5-js we have a prettier task - https://github.com/TBD54566975/web5-js/issues/266

should do something similar across all our repos so formatting stays in sync

Reply:
weird, thought this would have been taken care of by ktlint/detekt